arctic/store/_version_store_utils.py

```diff
@@ -28,9 +30,13 @@ def checksum(symbol, doc):
     Checksum the passed in dictionary
     """
     sha = hashlib.sha1()
-    sha.update(symbol)
-    for k in sorted(doc.iterkeys(), reverse=True):
-        sha.update(str(doc[k]))
+    sha.update(symbol.encode('ascii'))
```

Comment:
In python 2.X the default string encoding is ascii - therefore the conversion to ascii is a noop. 

In Python 3.X we have to encode to ascii to pass it into shalib. This means we don't support unicode string names, but we don't in 2.X already.